### PR TITLE
add a standalone bundle-size job

### PR DESCRIPTION
**Pull request by erwinmombay**
add a standalone job for the bundle size tracking which downloads both the nomodule and module builds

### CODE REVIEW

File: src/amp.js
Comment (kristoferbaxter) [inline src/amp.js:187]: Assuming this is unintentional?

File: .travis.yml
Comment (danielrozenberg) [inline .travis.yml:65]: How prevalent are these names in the JS community? (I just want to make sure that this isn't something we know but others would be confused about)

File: .travis.yml
Comment (danielrozenberg) [inline .travis.yml:49]: I think it's better to have the bundle size job in the `test` stage - as it currently stands, if any of the test or experiment jobs fail on a push build, this commit won't have generated bundle size data and will cause subsequent PRs to fail

/cc @rsimha for second opinion

File: build-system/pr-check/bundle-size.js
Comment (danielrozenberg) [inline build-system/pr-check/bundle-size.js:60]: I assume this was supposed to be `--on_pr_build`?

File: build-system/pr-check/bundle-size.js
Comment (danielrozenberg) [inline build-system/pr-check/bundle-size.js:62]: Aren't we using fancylog for these? 🤔

Comment (danielrozenberg) [inline build-system/pr-check/bundle-size.js:62]: Missing `gulp bundle-size --on_skipped_build` call

File: .travis.yml
Comment (rsimha) [reply .travis.yml:49]: +1. This can be part of the test stage.

File: .travis.yml
Comment (rsimha) [reply .travis.yml:65]: Fair point. Hope about calling the three jobs `Unminified Build`, `Nomodule Build`, and `Module Build`?

Comment (erwinmombay) [reply .travis.yml:65]: the JS community use the "module" and "nomodule" terminology pretty consistently. 

Alsoyes i think calling these "build" is more appropriate, will apply recommendation

File: src/amp.js
Comment (erwinmombay) [reply src/amp.js:187]: @kristoferbaxter sorry this was just to trigger an actual build test. will remove since it was tested

File: build-system/pr-check/bundle-size.js
Comment (erwinmombay) [reply build-system/pr-check/bundle-size.js]: done. switched to fancy-log in module-build.js, nomodule-build.js and bundle-size.js

File: build-system/pr-check/bundle-size.js
Comment (erwinmombay) [reply build-system/pr-check/bundle-size.js]: done.

File: build-system/pr-check/bundle-size.js
Comment (erwinmombay) [reply build-system/pr-check/bundle-size.js]: done. and added `await signalDistUpload('skipped');` right after

File: .travis.yml
Comment (erwinmombay) [reply .travis.yml]: done. does it matter what order it appears in the list?

File: .travis.yml
Comment (erwinmombay) [reply .travis.yml]: done.

File: src/amp.js
Comment (rsimha) [reply src/amp.js:187]: Reminder to get rid of this line.

File: build-system/pr-check/bundle-size.js
Comment (rsimha) [inline build-system/pr-check/bundle-size.js:61]: nit: Get rid of extra blank line for easy reading.

File: build-system/pr-check/bundle-size.js
Comment (rsimha) [inline build-system/pr-check/bundle-size.js:65]: nit: Same here.

File: .travis.yml
Comment (rsimha) [inline .travis.yml:74]: > does it matter what order it appears in the list?

The way you've done it LGTM. (This job will be the first to start in the test stage, so we'll see results ASAP.)

**Review by rsimha — APPROVED** [review]
Thanks for the cleanup and attention to detail! This PR LGTM after a couple minor comments below are addressed.

**Review by danielrozenberg — APPROVED** [review]

**Comment by rsimha** [comment]
Looks like this worked: https://github.com/ampproject/amphtml/pull/31025/checks?check_run_id=1381407330

Ship it!

**Pull request merged**
